languageserver/integration/resolvers.go

```diff
@@ -59,11 +59,16 @@ func deURI(path string) string {
 	return path
 }
 
+// isContractIdentifier returns true if the location string is a contract identifier (not a file path)
+func isContractIdentifier(locationString string) bool {
```

Comment:
Nit: Maybe rename this to `isContractFilename`, as this is checking for a filename extension.

In the Cadence codebase(s) "identifier" has the meaning of `[A-Za-z_][A-Za-z0-9_]*`